src/plugins/plot/chart/MctChart.vue

```diff
@@ -22,20 +22,22 @@
 
 <template>
   <div ref="chart" class="gl-plot-chart-area">
-    <canvas :style="canvasStyle" class="js-overlay-canvas"></canvas>
-    <canvas :style="canvasStyle" class="js-main-canvas"></canvas>
-    <div ref="limitArea" class="js-limit-area">
+    <canvas :style="canvasStyle" class="js-overlay-canvas" role="img" alt="2dContext"></canvas>
+    <canvas :style="canvasStyle" class="js-main-canvas" role="img" alt="webglContext"></canvas>
+    <div ref="limitArea" class="js-limit-area" aria-hidden="true">
       <limit-label
         v-for="(limitLabel, index) in visibleLimitLabels"
         :key="index"
         :point="limitLabel.point"
         :limit="limitLabel.limit"
+        role="text"
```

Comment:
This will be applied to the parent dom element of `LimitLabel`. We should do this inside of `LimitLabel.vue` instead I think.